www-apps/prowlarr/files/prowlarr.init

```diff
@@ -0,0 +1,7 @@
+#!/sbin/openrc-run
+
+pidfile="/run/${RC_SVCNAME}.pid"
```

Comment:
Personally, I prefer `/run/prolarr.pid` instead of `/run/${RC_SVCNAME}.pid`. Same for command_user below.